fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -67,11 +71,11 @@ const RollCallOpened = () => {
         <TextBlock text={STRINGS.roll_call_scan_description} />
         <QrReader
           delay={300}
-          onScan={handleScan}
+          onScan={handleScan} // {()=>handleScan} doesn't work
```

Comment:
Is it underlined in red ? If yes, try this:
```suggestion
          onScan={() => handleScan()}
```
You didn't type "handleScan()" in your comment, that's why I'm asking